### PR TITLE
[DOCKER] Fix Dockerfile.demo_android

### PR DESCRIPTION
**Pull request by tkat0**
Fixed [Deploy model on android, docker build error - Troubleshooting - TVM Discuss](https://discuss.tvm.ai/t/deploy-model-on-android-docker-build-error/7731)

After this fix, I was able to build the apk of `apps/android_deploy` in the Docker container.

### CODE REVIEW

File: docker/Dockerfile.demo_android [hunk 1]
```diff
@@ -47,10 +47,7 @@ RUN bash /install/ubuntu_install_androidsdk.sh
 COPY install/ubuntu_install_vulkan.sh /install/ubuntu_install_vulkan.sh
 RUN bash /install/ubuntu_install_vulkan.sh
 
-ENV VULKAN_SDK /usr/local/VulkanSDK/1.0.65.0/x86_64
-ENV PATH ${PATH}:${VULKAN_SDK}/bin
-ENV LD_LIBRARY_PATH ${LD_LIBRARY_PATH}:${VULKAN_SDK}/lib
-ENV VK_LAYER_PATH ${VULKAN_SDK}/etc/explicit_layer.d
+ENV VULKAN_SDK=/usr
```
Comment (tkat0) [inline docker/Dockerfile.demo_android:50]: Same as #5566

File: docker/install/ubuntu_install_androidsdk.sh [hunk 2]
```diff
@@ -68,20 +67,20 @@ extras;google;simulators
 extras;google;webdriver
 extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2
 extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2
-lldb;2.3
 platforms;android-26
 platforms;android-27
 tools
-ndk-bundle
+ndk;21.3.6528147
```
Comment (tkat0) [inline docker/install/ubuntu_install_androidsdk.sh:73]: `ndk-bundle` install the latest NDK version.
I specified the currently latest version.

File: docker/install/ubuntu_install_androidsdk.sh [hunk 3]
```diff
@@ -68,20 +67,20 @@ extras;google;simulators
...
 EOF
 
 mkdir /root/.android 2>/dev/null || true
 touch /root/.android/repositories.cfg
-(yes || true) | sdkmanager --licenses --sdk_root="$ANDROID_HOME"
+# NOTE: sdkmanager returns exit code 141
+(yes || true) | sdkmanager --licenses --sdk_root="$ANDROID_HOME" | true
```
Comment (tkat0) [inline docker/install/ubuntu_install_androidsdk.sh:79]: similar to https://github.com/facebook/flipper/pull/362

File: docker/install/ubuntu_install_androidsdk.sh [hunk 4]
```diff
@@ -68,20 +67,20 @@ extras;google;simulators
 extras;google;webdriver
 extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2
 extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2
-lldb;2.3
```
Comment (tkat0) [inline docker/install/ubuntu_install_androidsdk.sh:71]: I check with `sdkmanager --list`, I Deleted the package that does not currently exist.

File: docker/install/ubuntu_install_androidsdk.sh [hunk 5]
```diff
@@ -68,20 +66,20 @@ extras;google;simulators
...
 tools
-ndk-bundle
+ndk;21.3.6528147
 EOF
 
 mkdir /root/.android 2>/dev/null || true
 touch /root/.android/repositories.cfg
-(yes || true) | sdkmanager --licenses --sdk_root="$ANDROID_HOME"
+# NOTE: sdkmanager returns exit code 141
+(yes || true) | sdkmanager --licenses --sdk_root="$ANDROID_HOME" || true
```
Comment (tkat0) [inline docker/install/ubuntu_install_androidsdk.sh:78]: similar to facebook/flipper#362

**Comment by tqchen** [comment]
THanks @tkat0 !

**Pull request merged**